client/src/utils/index.ts

```diff
@@ -41,3 +41,29 @@ export function get(path: string, params = {}) {
     throw new Error(`${res.statusText} (${res.status})`);
   };
 }
+
+
+export function convertBytesToMb(b: number): number {
+  let gbValue = (b / (1000 * 1000));
+  return gbValue
+}
+
+export function convertBytesToGb(b: number): number {
+  let gbValue = (b / (1000 * 1000 * 1000));
+  return gbValue
+}
+
+export function convertKbToTb(b: number): number {
+  let gbValue = (b / (1000 * 1000));
+  return gbValue
+}
+
+export function convertGbToTb(b: number): number {
+  let gbValue = (b / (1000));
+  return gbValue
+}
+
+export function convertTbToPb(b: number): number {
+  let gbValue = (b / (1000 * 1000 ));
+  return gbValue
+}
```

Comment:
Did you want to do something like this here? https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable-string

Reply:
Yes and no! So yes I need a way to convert from different base units, some are returned from Prometheus as Bytes/Kb thus far. I needed it in raw format without the string label for chart series data, but also need the stats to show a label. Would be great to improve it with this and have a single dynamic conversion function that outputs forms for both situations. Not sure about unit standardization out of Prometheus, need to investigate more. Thank you!